disk/disk_unix.go

```diff
@@ -2,11 +2,11 @@
 
 package disk
 
-import "syscall"
+import "golang.org/x/sys/unix"
 
 func Usage(path string) (*UsageStat, error) {
-	stat := syscall.Statfs_t{}
-	err := syscall.Statfs(path, &stat)
+	stat := unix.Statfs_t{}
```

Comment:
I'm sorry, but could you tell me why use `unix` instead of `syscall`?
If use `unix`, at least darwin platform this error occured.

```
# github.com/shirou/gopsutil/disk
./disk_unix.go:17: cannot use stat (type unix.Statfs_t) as type syscall.Statfs_t in argument to getFsType
```

`getFsType` is this function 

```
func getFsType(stat syscall.Statfs_t) string {
        return common.IntToString(stat.Fstypename[:])
}
```